pinning-services/index.js

```diff
@@ -199,6 +249,21 @@ async function getDatSeed(datSeedName, projName, domain, recordName) {
   }
 }
 
+function addDomainAccountDigitalOcean(domain, doToken) {
+  const url = `https://api.digitalocean.com/v2/domains`;
+  const headers = {
+    'Content-Type': 'application/json',
+    'Authorization': `Bearer ${doToken}`
+  };
+  const data = JSON.stringify({
+    name: domain
+  });
+  console.log(`POST ${url} with data ${data}`);
+  return fetch(url, { method: 'POST', headers: headers, body: data })
```

Comment:
Do you know what happens if it alrdy exists?

Reply:
yes returns an error in json. Also there is no json['domain'] set which is used to return the true/false of command on line 264 and then used to throw an exception in 285

Reply:
Just wanna make sure it doesn't overwrite existing...